.env.example

```diff
@@ -1,25 +1,27 @@
-NEXT_PUBLIC_URL_SPID_LOGIN=http://localhost:9090/login
-NEXT_PUBLIC_URL_IO=https://io.italia.it
-
-# TOKEN SPID LEVEL VALUE
-NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1=https://www.spid.gov.it/SpidL1
-NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L2=https://www.spid.gov.it/SpidL2
-NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L3=https://www.spid.gov.it/SpidL3
-
-NEXT_PUBLIC_API_BASE_URL=http://localhost:7071
-NEXT_PUBLIC_API_BASE_PATH=/api/v1
-NEXT_PUBLIC_API_FETCH_TIMEOUT=5000
-NEXT_PUBLIC_API_FETCH_MAX_RETRY=5
-
-NEXT_PUBLIC_FEATURE_FLAG=false
-NEXT_PUBLIC_SPID_CIE_ENTITY_ID=xx_servizicie
-
-NEXT_PUBLIC_ANALYTICS_ENABLE=true
-NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
-NEXT_PUBLIC_ANALYTICS_TOKEN="" # FIXME: JIRA TICKET 862
-NEXT_PUBLIC_ANALYTICS_API_HOST="" # FIXME: JIRA TICKET 862
-NEXT_PUBLIC_ANALYTICS_PERSISTENCE="" # FIXME: JIRA TICKET 862
-NEXT_PUBLIC_ANALYTICS_LOG_IP=true
-NEXT_PUBLIC_ANALYTICS_DEBUG=true
-
-NEXT_PUBLIC_SPID_TEST_ENV_ENABLED=true
\ No newline at end of file
+NEXT_PUBLIC_URL_SPID_LOGIN=http://localhost:9090/login
+NEXT_PUBLIC_URL_IO=https://io.italia.it
+
+# TOKEN SPID LEVEL VALUE
+NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1=https://www.spid.gov.it/SpidL1
+NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L2=https://www.spid.gov.it/SpidL2
+NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L3=https://www.spid.gov.it/SpidL3
+
+NEXT_PUBLIC_API_BASE_URL=http://localhost:7071
+NEXT_PUBLIC_API_BASE_PATH=/api/v1
+NEXT_PUBLIC_API_FETCH_TIMEOUT=5000
+NEXT_PUBLIC_API_FETCH_MAX_RETRY=5
+
+NEXT_PUBLIC_FEATURE_FLAG=false
+NEXT_PUBLIC_SPID_CIE_ENTITY_ID=xx_servizicie
+
+NEXT_PUBLIC_ANALYTICS_ENABLE=true
+NEXT_PUBLIC_ANALYTICS_MOCK=true # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_TOKEN="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_API_HOST="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_PERSISTENCE="" # FIXME: JIRA TICKET 862
+NEXT_PUBLIC_ANALYTICS_LOG_IP=true
+NEXT_PUBLIC_ANALYTICS_DEBUG=true
+
+NEXT_PUBLIC_SPID_TEST_ENV_ENABLED=true
+
+NEXT_PUBLIC_FOOTER_PRODUCT_LIST=https://dev.selfcare.pagopa.it/assets/products.json
```

Comment:
Why I see all this changes? Maybe different IDEs encoding of text files?

Reply:
During PC env reinstallation I forgot to set LF instead of CRLF inside VS code